### DISCUSSION TITLE
Is this library dead?

### DISCUSSION THREAD

**Discussion by darrenhull** [aztfmod/terraform-azurerm-caf#1964] [General] [answered]
I'm a bit concerned with the lack of traction and support on this library. There is no real feedback on discussions and its hard to see where its going. 

One example is that In March 2022 the new version of Azure front door was released yet this is not supported in this library. Is there a plan to implement it or a roadmap showing when it would be implemented. Now Azure Front door is just one of many things. 
I get the lack of resources but I see people offering to help with PR's and those PR's are just left. If the library is so far behind whats available in azure or azurerm then its got to be time to realise that this pattern doesn't work and maybe a thinner wrapper is needed?

Hopefully this generates a discussion...

Ah... I have just seen this which answers my question: 
⚠️ This solution, offered by the Open-Source community, will no longer receive contributions from Microsoft. Customers are encouraged to transition to [Microsoft Azure Verified Modules](https://aka.ms/avm) for Microsoft support and updates.

**Comment by chatelain-io** [comment]
I would have liked to know that they were transitioning to Azure Verified Modules a couple of months ago in favour of this project (or other project served by Microsoft).

At the time, AVM was still new and featureless, and they didn't communicate that would be the next step, so I went with this project since it had a history, active and way more features than AVM.

I had to fork the project to make sure I could make the adjustment in time also got hit by the performance issues, bad implementation, etc.  

From my point of view, I would suggest to not use this project, check if AVM is a good fit for you, but it is still lacking in feature. For myself, I've started transitioning to Crossplane. They have pretty neat providers, and you can still use Terraform on the edge case.

**Comment by arnaudlh** [comment, marked as answer]
If you are looking for Microsoft-supported module, yes go AVM. 
This project will remain open-source and community-based, fulfilling a different feature set and an infra-as-config approach. 
We also are working toward more community inclusion in the project maintainers, feel free to reach out to discuss!

**Reply by jtdossett** [reply]
Arnaud, I am interested, what's the best way to reach out?

**Reply by arnaudlh** [reply]
Lets start with LinkedIn!

**Reply by darrenhull** [reply]
@arnaudlh i'm also interested and have reached out on LinkedIn.

**Reply by darrenhull** [reply]
i was also wondering if in time this module could swap out the azurerm modules for the verified modules.
